frontend/src/features/splReports/containers/SplReportContainer.tsx

```diff
@@ -33,17 +46,52 @@ const SplReportContainer: React.FunctionComponent<ISplReportContainerProps> = ()
     updateProjectReport,
     exportReport,
   } = useProjectSnapshotApi();
+  const [snapshotFilter, setSnapshotFilter] = React.useState<ISnapshotFilter>({
+    projectNumber: '',
+    agency: '',
+    fiscalYear: '',
+    sortBy: { projectId: 'asc', snapshotOn: 'desc' },
+  });
 
   const id = currentReport?.id;
   useEffect(() => {
     const fetch = async () => {
       if (id) {
-        const data = await getProjectReportSnapshotsById(id);
+        let data = await getProjectReportSnapshotsById(id);
```

Comment:
/nit would prefer to always use descriptive variable names where possible. So in this case snapshots or projectReportSnapshots is preferred over just data (already had issues in the past with naming conflicts between data and the axios data object from axios responses.